.idea/olx2.iml

```diff
@@ -3,6 +3,7 @@
   <component name="NewModuleRootManager">
```

Comment:
.idea should be in gitignore. Please exclude this file from commit

Reply:
Done